airflow/cli/cli_parser.py

```diff
@@ -142,6 +142,10 @@ def positive_int(value):
     ("-e", "--end-date"),
     help="Override end_date YYYY-MM-DD",
     type=parsedate)
+ARG_OUTPUT_PATH = Arg(
+    ("-o", "--output-path",),
+    help="The output for generated yaml files",
+    default="/tmp/airflow_yaml_output/")
```

Comment:
Wouldn't it be better to write in Current Working Directory? This will provide easier access to these files. Most linux utilities use CWD by default.
```
tar xf pathtoarchive/archive.tar

```

Reply:
By putting it in /tmp we can assure that it will eventually be deleted as the system needs space.

Reply:
I understand the reasons, but I'm afraid it might be more difficult to use because of this.  It is common to save files in the current directory by default.  If the user wishes to write to tmp, he can execute the following sequence of commands.
```bash
cd /tmp/
mkdir -p airflow_tmp
airflow dags generate_kubernetes_pod_yaml example_bash_operator
ls
```
If we save in / tmp / the user may feel confused. When I unpack or create a file, it expects it to be visible after executing `` ls ''
```bash
cd ~
airflow dags generate_kubernetes_pod_yaml example_bash_operator
ls
# Generated files not visible.
```
In 2020, users have large enough hard drives that we do not have to worry about saving too much data. I think their convenience and productivity are more important.

Reply:
@kaxil WDYT?

Reply:
Valid point

Reply:
I would personally be really annoyed if I ran this command and then suddenly had to clean up 20 yaml files in my current directory

Reply:
I agree with Kamil on thjs

Reply:
@potiuk WDYT

Reply:
You could output to a directory in the current directory. That way you just need to delete the folder if that is the concern

Reply:
i.e the subcommand can create a folder example "output_yaml_files" or something in the current directory that stores all the yaml's

Reply:
Done :)